doc/api/signed-requests.md

```diff
@@ -0,0 +1,127 @@
+Signed Requests
+===============
+
+Whenever you use these access codes, you are required to make signed requests.  This process ensures that the request is coming from an authenticated user and prevents replay attacks.  You will transmit the public `code` and use the private `secret` to generate the signature.
+
+The signature process is broken down into small steps for this explanation.  First, you create a date stamp.  Next, some signed headers are formatted.  Next, some content that will be signed.  The [HMAC] of the content is calculated.  Lastly, the request has the right headers set.
```

Comment:
Can the third step be written as:

After that, the content will need to be signed and an [HMAC] of the signed content calculated.